src/components/Modals/SettingsModal/SettingsModal.vue

```diff
@@ -1,76 +1,75 @@
 <template>
   <v-dialog
     v-model="dialog"
+    max-width="800px"
     scrollable
+    :content-class="isPhone ? 'rounded-0' : 'rounded-form'"
     :width="dialogWidth"
-    :fullscreen="phoneLayout"
+    :fullscreen="isPhone"
   >
-    <v-card style="min-height: 400px; overflow: hidden !important">
```

Comment:
I'm not sure about removing the dedicated min-height... on desktop the modal height now jumps around on different tabs and that's pretty annoying. Issues were opened in the past for this very issue.
Same goes for other modals as well

Reply:
It doesn't seem to be reproduced after fixing the layout.